lib/model/model.go

```diff
@@ -1926,8 +1921,8 @@ func (m *Model) internalScanFolderSubdirs(ctx context.Context, folder string, su
 
 	for f := range fchan {
 		if len(batch) == maxBatchSizeFiles || batchSizeBytes > maxBatchSizeBytes {
-			if err := m.CheckFolderHealth(folder); err != nil {
-				l.Infof("Stopping folder %s mid-scan due to folder error: %s", folderCfg.Description(), err)
+			if err := runner.CheckHealth(); err != nil {
+				l.Debugln("Stopping scan of folder %s due to: %s", folderCfg.Description(), err)
```

Comment:
We already log this in info or higher, so I'd just remove this

Reply:
Agreed, the only added information is, that it happened during scan, which should also be clear from context.

Reply:
you can do CheckHealth("during scan") or something to explain where the health check is happening, and then log that.

Reply:
Actually, we do not necessarily log anything on info or higher unless the error is new or changed. So in a debug scenario, I think it is valid that we log the error anyway and from the position of origin.

Reply:
Fine whatever.